src/features/projectsV2/ProjectDetails/components/OtherInterventionInfo.tsx

```
@@ -0,0 +1,141 @@
+      return result;
+    }
+  }, [plantLocationInfo]);
+
+  const shouldDisplayImageCarousel =
+    sampleInterventionSpeciesImages !== undefined &&
+    sampleInterventionSpeciesImages?.length > 0;
+
+  const checkForPublicData = plantLocationInfo?.metadata.public.length !== 0
+
+  const content = [
+    <InterventionHeader plHid={plantLocationInfo?.hid} type={plantLocationInfo?.type} />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)